### PR TITLE
Update test data

### PR DESCRIPTION
**Pull request by snnn**
**Description**: 

**Motivation and Context**
- Why is this change required? What problem does it solve?

From: Hari Seshadri 
Sent: Thursday, September 19, 2019 6:30 PM
To: Changming Sun 
Subject: RE: A recent onnxruntime change breaks mask rcnn onnx model (converted by keras)

This is Keras mask RCNN. Could you please help include it in our test folder zip ? Wenbing/David can provide the model.

- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

File: onnxruntime/test/onnx/main.cc [hunk 1]
```diff
@@ -530,6 +530,7 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
   broken_tests.insert({"tf_mobilenet_v2_1.4_224", "result mismatch"});
   broken_tests.insert({"tf_mobilenet_v1_1.0_224", "result mismatch"});
   broken_tests.insert({"mobilenetv2-1.0", "result mismatch"});
+  broken_tests.insert({"candy", "result mismatch"});
```
Comment (jiafatom) [inline onnxruntime/test/onnx/main.cc]: Synced offline, this change also contains models other than keras mask rcnn.

File: onnxruntime/test/onnx/main.cc [hunk 2]
```diff
@@ -563,6 +564,7 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
 
 #ifdef USE_CUDA
   broken_tests.insert({"mxnet_arcface", "result mismatch"});
+  broken_tests.insert({"mask_rcnn_keras", "result mismatch"});
```
Comment (jiafatom) [inline onnxruntime/test/onnx/main.cc]: Are we going to investigate this mismatch?

File: tools/ci_build/github/azure-pipelines/templates/set-test-data-variables-step.yml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # sets variables $(TestDataUrl) and $(TestDataChecksum)
 
 parameters:
-  TestDataUrl: https://onnxruntimetestdata.blob.core.windows.net/models/20190729.zip
-  TestDataChecksum: ef36198a9ebaca8413b9739b15e87954
+  TestDataUrl: https://onnxruntimetestdata.blob.core.windows.net/models/20190925.zip
+  TestDataChecksum: ae5fb5e3dd5e4937c8343d13e3be680c
```
Comment (jiafatom) [inline tools/ci_build/github/azure-pipelines/templates/set-test-data-variables-step.yml]: This test data url contains keras mask rcnn, right?

Comment (snnn) [reply tools/ci_build/github/azure-pipelines/templates/set-test-data-variables-step.yml]: Yes.

File: onnxruntime/test/onnx/main.cc (hunk 2 above)
Comment (snnn) [reply onnxruntime/test/onnx/main.cc]: Sure. We'll discuss it in today's scrum.

**Pull request merged**